apps/api/src/domain/domainModelValidators/__tests__/domainModelValidators/buildDomainModelValidatorTestCases/spatial-feature.domainModelValidatorTestCase.ts

```diff
@@ -1,53 +1,6 @@
-import InvalidGeometryTypeForSpatialFeatureError from '../../../../models/spatial-feature/errors/InvalidGeometryTypeForSpatialFeatureError';
```

Comment:
It's worth going through the `domainModelValidators` test directory and renaming whatever remains, since the test itself is no longer.

Reply:
done